x-pack/elastic-agent/pkg/agent/application/pipeline/stream/operator_stream.go

```diff
@@ -44,8 +48,17 @@ func (b *operatorStream) Specs() map[string]program.Spec {
 	return nil
 }
 
-func (b *operatorStream) Execute(cfg configrequest.Request) error {
-	return b.configHandler.HandleConfig(cfg)
+func (b *operatorStream) Execute(ctx context.Context, cfg configrequest.Request) (err error) {
+	// TODO: double check `route` as name
```

Comment:
`*operatorStream.Execute()` is only being called by `*router.Route`, so I'm wondering if that would be the right span name, or something else.